src/components/bottom-sheet/bottom-sheet.tsx

```diff
@@ -12,6 +12,26 @@ export default class BottomSheet extends PureComponent<IBottomSheetProps> {
     this.onExit = this.onExit.bind(this);
   }
 
+  public componentDidMount(): void {
+    if (this.props.show && typeof window !== "undefined") {
+      document.body.classList.add("q-disable-scroll");
+    }
+  }
+
```

Comment:
Please also remove this class in componentWillUnMount life cycle

Reply:
Resolved